python/ray/train/_internal/state/schema.py

```diff
@@ -5,13 +5,17 @@
 from ray.dashboard.modules.job.pydantic_models import JobDetails
 from ray.util.annotations import DeveloperAPI
 
+MAX_ERROR_STACK_TRACE_LENGTH = 3000
```

Comment:
let's do 50,000 characters to start. We can be very lenient I think. 50k chars should be ~50kb.

Reply:
OK, updated!